### DISCUSSION TITLE
bigfix/10.0/mcm/welcome/BigFix_MCM_welcome

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#22] [Q&A]
# bigfix/10.0/mcm/welcome/BigFix_MCM_welcome

Welcome to the Modern Client Management and BigFix Mobile documentation, where you can find information about how to install, maintain, and use Modern Client Management and BigFix Mobile.

https://help.hcltechsw.com/bigfix/10.0/mcm/welcome/BigFix_MCM_welcome.html

**Comment by gayathripg** [comment]
Test
